nox/tox_to_nox.py

```diff
@@ -37,7 +51,7 @@ def wrapjoin(seq: Iterator[Any]) -> str:
 
 def fixname(envname: str) -> str:
     """Replace dashes with underscores and check if the result is a valid identifier."""
-    envname = envname.replace("-", "_")
+    envname = envname.replace("-", "_").replace("testenv:", "")
```

Comment:
I'm not sure why this is needed, and if it would have a negative effect on old tox (tox <= 3)

Reply:
Verified purpose.